docs/Primer.MD

```diff
@@ -249,9 +249,9 @@ Is transformed to:
 data class NadelHydrationFieldInstruction(
     override val location: FieldCoordinates, // User.friend
     val hydratedFieldDef: GraphQLFieldDefinition, // Schema definition for User.friend
-    val actorService: Service, // Users service i.e. service owning users.friendById
-    val queryPathToActorField: NadelQueryPath, // users.friendById
-    val actorFieldDef: GraphQLFieldDefinition, // Schema definition for users.friendById
+    val backingService: Service, // Users service i.e. service owning users.friendById
+    val queryPathToBackingField: NadelQueryPath, // users.friendById
+    val backingFieldDef: GraphQLFieldDefinition, // Schema definition for users.friendById
     val actorInputValueDefs: List<NadelHydrationActorInputDef>, // [{ name: "ids" value: "$source.friendId" }]
```

Comment:
what about this actorInput? Does not make sense to rename to backingInputValueDefs?

Reply:
Updated, thanks for pointing that out